src/providers/fault-store/fault-store.ts

```diff
@@ -52,7 +52,7 @@ export class FaultStoreProvider {
 
     this.faultActions.loadFaults();
     this.lastFault$ = this.store.select((state) => state.faults.lastFault);
-    this.currentfaults$ = this.store.select((state) => state.faults);
+    this.currentFaults$ = this.store.select((state) => state.faults);
```

Comment:
I think this also need to be changed in all the components that subscribe to this.currentFaults$ ?

Reply:
👍 Yup, have just done a global find and replace. Cheers